### DISCUSSION TITLE
Termux (aarch64), CLI project management solution, for coding ptoject and wider auspice

### DISCUSSION THREAD

**Discussion by b9Joker108** [GothenburgBitFactory/taskwarrior#3304] [Q&A]
Hi

I am prospecting a free and open-source software development project management application that is installable in Termux (aarch64), usable in the CLI, and can be integrated with GitHub, to manage a coding project, and other aspects of the project auspice. I understand that Taskwarror, endeavours to be best-in-class, in relation to this usecase. 

I just installed taskwarrior (2.6.2-3), and couldn't get the new install to execute, or find help instructions reguarding it, in the CLI. So, I found my way to the website, X profile, then GitHub repository. I noted that there is a very new breaking version 3.0, that is very recent, circa four (4) days ago. This version hasn't yet been packaged for Termux.

Should I start using 2.6.2-3, build install 3.0, or wait for 3.0 to be packaged for Termux?

Any assistance is greatly appreciated.

Thanking you in anticipation
Beauford

**Comment by djmitche** [comment]
I also use Taskwarrior on termux, and I don't know what the situation is for packaging the new release in termux.
